docs/sources/setup-grafana/configure-security/configure-authentication/saml/index.md

```diff
@@ -373,6 +373,20 @@ SAML's single logout feature allows users to log out from all applications assoc
 `HTTP-Redirect` and `HTTP-POST` bindings are supported for single logout.
 When using `HTTP-Redirect` bindings the query should include a request signature.
 
+#### Configure single logout
+
+{{% admonition type="note" %}}
+Proper `NameID` and `SessionIndex` support is available in public preview behind the `improvedExternalSessionHandling` feature toggle, starting from Grafana v11.5.
+{{% /admonition %}}
+
+Before Grafana version 11.5, the `Login` attribute value (extracted from the SAML assertion using the `assertion_attribute_login` configuration) was used as the `NameID` in the logout request. This could cause issues with single logout if the `assertion_attribute_login` value differed from what the Identity Provider (IdP) expected.
```

Comment:
If you decide to do the above mentioned suggestions, the `Configure single logout` section can be edited to something like:

```
To configure single logout in Grafana:
	1.	Enable the `single_logout option` in your configuration.
	2.	Ensure the `name_id_format` matches the format your IdP expects (e.g., `urn:oasis:names:tc:SAML:2.0:nameid-format:transient`).
	3.	Enable the `improvedExternalSessionHandling` feature toggle for full NameID and SessionIndex support (Grafana v11.5+).
```